modules/nf-core/cadd/cadd.diff

```diff
@@ -0,0 +1,14 @@
+Changes in module 'nf-core/cadd'
+--- modules/nf-core/cadd/main.nf
++++ modules/nf-core/cadd/main.nf
+@@ -43,7 +43,7 @@
+     def prefix = task.ext.prefix ?: "${meta.id}"
+     def VERSION = "1.6.post1" // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+     """
+-    touch ${prefix}.tsv.gz
++    echo "" | gzip > ${prefix}.tsv.gz
+ 
+     cat <<-END_VERSIONS > versions.yml
+     "${task.process}":
+
+************************************************************
```

Comment:
are these changes that we should try to add to the nf-core module as well?

Reply:
Probably, I'll remove them from here for now.